bin/env.sh

```diff
@@ -4,7 +4,7 @@ BASEDIR=`dirname $0`/..
 
 if [ ! -d "$BASEDIR/env" ]; then
 
-    virtualenv -q $BASEDIR/env --prompt='(dcos) '
+    virtualenv -p python3 -q $BASEDIR/env --prompt='(dcos) '
```

Comment:
It's currently broken on the ubuntu docker we're using: https://bugs.launchpad.net/ubuntu/+source/python3.4/+bug/1290847

I've failed an issue #670